ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+# ISSUE_TEMPLATE
```

Comment:
I think the title should be `# Description`, Instead of `# ISSUE_TEMPLATE`. You can test this once the code is merged.

Reply:
Thanks for the comment, I will modify it.